src/backend/distributed/relay/relay_event_utility.c

```diff
@@ -41,14 +42,23 @@
 #include "utils/palloc.h"
 #include "utils/relcache.h"
 
+/* expression tree walker context for rewriting row references */
```

Comment:
Did we really? Where? I'd actually love to go through and move all non-interface-related structs into implementation files, then…